experimental/dwarf.go

```diff
@@ -0,0 +1,22 @@
+package experimental
+
+import (
+	"context"
+)
+
+type enableDWARFBasedStackTraceKey struct{}
+
+// WithDWARFBasedStackTrace enables the DWARF based stack traces in the face of runtime errors.
```

Comment:
does this invalidate or otherwise make complication cache tricky? ex if the cache was built with this off and used when on or visa versa?

Reply:
good point, will do it in another pr